src/schedule.md

```diff
@@ -3,12 +3,21 @@ layout: default
 title: Schedule
 ---
 
+<p>The conference will run from Friday 22nd till Monday 25th September 2023.</p>
+
+<p>The conference will be made up of 3 days of talks and workshops from Friday - Sunday and a single day of sprints on Monday. All ticket holders are entitled to join the sprints day at no additional charge.</p>
+
+<p>We will be holding a Young Coder’s day on Saturday 23rd September suitable for young people aged from about 8 years old to 16 years old. Young Coder’s tickets are available at £6 for a young coder and an accompanying adult. The adult will accompany and be responsible for their young coder for the duration of the day.</p>
+
+<p>We will also be holding a Django Girls workshop on Saturday 23rd September. This workshop will have it’s own application process which we will update here shortly. It will not be a general access workshop within the the broader conference programme.</p>
```

Comment:
nitpick: its rather than it's
```suggestion
<p>We will also be holding a Django Girls workshop on Saturday 23rd September. This workshop will have its own application process which we will update here shortly. It will not be a general access workshop within the the broader conference programme.</p>
```